src/geointerface.jl

```diff
@@ -23,57 +35,282 @@ let pointtypes = (wkbPoint, wkbPoint25D, wkbPointM, wkbPointZM),
         wkbGeometryCollectionZM,
     )
 
-    function GeoInterface.geotype(g::AbstractGeometry)::Symbol
-        gtype = getgeomtype(g)
+    GeometryTraits = Union{
+        GeoInterface.PointTrait,
+        GeoInterface.MultiPointTrait,
+        GeoInterface.LineStringTrait,
+        GeoInterface.LinearRingTrait,
+        GeoInterface.MultiLineStringTrait,
+        GeoInterface.PolygonTrait,
+        GeoInterface.MultiPolygonTrait,
+        GeoInterface.GeometryCollectionTrait,
+        GeoInterface.CircularStringTrait,
+        GeoInterface.CompoundCurveTrait,
+        GeoInterface.CurvePolygonTrait,
+        GeoInterface.MultiSurfaceTrait,
+        GeoInterface.PolyhedralSurfaceTrait,
+        GeoInterface.TINTrait,
+        GeoInterface.TriangleTrait,
+    }
+
+    # Feature
+    GeoInterface.isfeature(feat::AbstractFeature) = true
+    function GeoInterface.properties(feat::AbstractFeature)
+        return (;
```

Comment:
This is a little confusing, maybe refactor with named variables?

Reply:
Resolved by https://github.com/yeesian/ArchGDAL.jl/pull/290/commits/3079fd31974a6ef5ccbf4241ee4b5bc2c4bfca2b